.env.development

```diff
@@ -0,0 +1,34 @@
+# AWS / LocalStack Config
+AWS_ACCESS_KEY_ID=test
+AWS_SECRET_ACCESS_KEY=test
+AWS_DEFAULT_REGION=us-east-1
+S3_BUCKET_NAME=biotablero-local
+RDS_HOST=localhost
+RDS_PORT=5433
+RDS_DB_NAME=biotablero-db
+RDS_USER=postgres
+RDS_PASSWORD=2a2f56b0
```

Comment:
Avoid sharing passwords and database connection data directly in the code. If necessary, temporarily create a shared document (e.g., Google Docs) to store credentials and other sensitive information.